src/main/java/binnie/extrabees/gui/database/ControlProductsItem.java

```diff
@@ -24,7 +24,7 @@ public ControlProductsItem(ControlList<ControlProductsBox.Product> controlList,
 		}
 
 		item.setItemStack(value.item);
-		float time = (int) (55000.0 / value.chance);
+		float time = (int) (550.0 / value.chance);
```

Comment:
I checked it bug on old version of code after decompile and port to Forestry 4.2.
The problem is the place to be.